tools/qlog/src/lib.rs

```diff
@@ -1349,26 +1350,26 @@ pub enum EventData {
     // ================================================================== //
     // RECOVERY
     RecoveryParametersSet {
-        reordering_threshold: Option<u64>,
-        time_threshold: Option<u64>,
-        timer_granularity: Option<u64>,
-        initial_rtt: Option<u64>,
+        reordering_threshold: Option<u16>,
+        time_threshold: Option<f32>,
+        timer_granularity: Option<u16>,
+        initial_rtt: Option<f32>,
 
-        max_datagram_size: Option<u64>,
+        max_datagram_size: Option<u32>,
         initial_congestion_window: Option<u64>,
-        minimum_congestion_window: Option<u64>,
-        loss_reduction_factor: Option<u64>,
-        persistent_congestion_threshold: Option<u64>,
+        minimum_congestion_window: Option<u32>,
```

Comment:
Since initial_congestion_window is u64, I think we also use u64 here for consistency.

Reply:
Taken straight from https://quicwg.org/qlog/draft-ietf-quic-qlog-quic-events.html#name-parameters_set-2